client-py/iotdb/SessionPool.py

```diff
@@ -18,32 +18,30 @@
 import logging
 import multiprocessing
 import time
-from multiprocessing import Queue
-from threading import Lock
+from queue import Queue
```

Comment:
`queue.Queue` is used for multi-threading, and `multiprocessing.Queue` is used for multi-processing. `threading.Lock` is used for multi-threading, and `multiprocessing.Lock` is used for multi-processing.  I think we should use `queue.Queue` and `threading.Lock` if the session-pool is designed for the usage in multi-threading environment, else we should use `multiprocessing.Queue` and `multiprocessing.Lock` if it's designed for the usage of multi process environment.